.buildkite/pipeline.yml

```diff
@@ -0,0 +1,7 @@
+steps:
+  - label: ":wrench: Build"
+    command:
+    - "nvm install"
```

Comment:
why do we need `nvm install`?

Reply:
you can control which node version to use in '.nvmrc' file. Before each build that node will be installed

Reply:
this project is pure iOS, so I guess we can remove this